plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraSession.java

```diff
@@ -199,7 +198,7 @@ public CassandraTable getTable(SchemaTableName schemaTableName)
 
             // column ordering
             List<ExtraColumnMetadata> extras = extraColumnMetadataCodec.fromJson(columnOrderingString);
-            List<String> explicitColumnOrder = new ArrayList<>(ImmutableList.copyOf(transform(extras, ExtraColumnMetadata::getName)));
+            List<String> explicitColumnOrder = extras.stream().map(ExtraColumnMetadata::getName).collect(toList());
```

Comment:
toImmutableList

Reply:
The `explicitColumnOrder` is modified 3 statements below

Reply:
It’s improper to modify the list returned by toList(). There. Are no guarantees that such list is immutable or safe to modify. If the code need to modify it, it should create an ArrayList (or similar) out of the result or use a custom collector

Reply:
Actually yeah, it is explicitly mentioned that 

> There are no guarantees on the type, mutability, serializability, or thread-safety of the {@code List} returned;

Let me update this call site